src/patternfly/components/ContextSelector/context-selector-toggle.hbs

```diff
@@ -7,5 +7,10 @@
   {{#if context-selector-toggle--attribute}}
     {{{context-selector-toggle--attribute}}}
   {{/if}}>
+  {{#if context-selector-toggle--icon}}
```

Comment:
Doesn't look like this var is ever used - do you need it?

Reply:
Nope, I'll remove it! Left over from trying to get the plain variant with kebab icon.